clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -1089,6 +1089,8 @@ message IndexLiveSettings {
     google.protobuf.Int32Value defaultTerminateAfter = 13;
     // Merge precopy would be stopped after this time, or 0 for no checks, default: 0
     google.protobuf.UInt64Value maxMergePreCopyDurationSec = 14;
+    // Collect and publish additional index metrics, default: false
```

Comment:
Should the commend indicate this may introduce performance penalty if set to `true`?